python/lancedb/pydantic.py

```diff
@@ -238,27 +238,18 @@ def pydantic_to_schema(model: Type[pydantic.BaseModel]) -> pa.Schema:
     >>> from typing import List, Optional
     >>> import pydantic
     >>> from lancedb.pydantic import pydantic_to_schema
-    ...
-    >>> class InnerModel(pydantic.BaseModel):
-    ...     a: str
-    ...     b: Optional[float]
```

Comment:
Does having an inner model not work?

Reply:
It works fine in the unit test but for whatever reason the same code snippet in doctests don't work